azkaban-db/src/main/sql/create.version_set.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE version_set
```

Comment:
image_version_set table already added as part of the PR: https://github.com/azkaban/azkaban/pull/2651

Reply:
That is not in the master branch. Second, I think the schema we anyway have to change as created_on and created_by is not required with this definition of the version set.